integration/base/src/main/java/com/inrupt/client/integration/base/Utils.java

```diff
@@ -166,6 +173,96 @@ public static Set<Triple> publicAgentPolicyTriples(final URI acl) {
         return triples;
     }
 
+    public static void createPublicContainer(final SolidSyncClient authClient, final URI publicContainerURI) {
+        if (!exists(authClient, publicContainerURI)) {
+            try (SolidContainer newContainer = new SolidContainer(publicContainerURI)) {
+                try (final SolidContainer container = authClient.create(newContainer)) {
+                    container.getMetadata().getAcl().ifPresent(acl -> {
+                        int i = 0; // in case we cannot read the acl, we try 2 times before giving up
```

Comment:
This issue this method is fixing is (I believe) symptomatic of some other issue, and it would be better to fix the underlying issue.

For example, the ACL is not present because another process has just deleted the corresponding resource.